platforms/documentation/docs/src/docs/userguide/authoring-builds/introduction/build_lifecycle_intermediate.adoc

```diff
@@ -0,0 +1,116 @@
+// Copyright (C) 2024 Gradle, Inc.
+//
+// Licensed under the Creative Commons Attribution-Noncommercial-ShareAlike 4.0 International License.;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[build_lifecycle]]
+= Gradle Build Lifecycle
+
+The build lifecycle is the sequence of phases Gradle executes to turn your build scripts into completed work, from initializing the build environment to configuring projects and finally executing tasks.
```

Comment:
I'm not sure about the phrasing here - "turn your build scripts into completed work". Did you get this from existing documentation somewhere?

Reply:
It's also the wording that's been on an equivalent page elsewhere in the docs.